src/main/java/duke/Deadline.java

```diff
@@ -0,0 +1,59 @@
+package duke;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Deadline class represents a task with a specified deadline.
+ */
+public class Deadline extends Task {
+    private LocalDate date;
+
+    /**
+     * Constructor for Deadline object.
+     * @param isComplete Boolean to indicate if deadline is complete
+     * @param index Index position of Task in TaskList
+     * @param instructions Task description
+     * @param date Deadline
+     */
```

Comment:
Should leave empty line between descriptions and parameters. Same for the rest of JavaDoc comments.